modules/transports/optional/fix/src/test/java/org/apache/synapse/transport/fix/FIXTransportSenderInitTest.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.synapse.transport.fix;
+
+import junit.framework.TestCase;
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.axis2.description.AxisService;
+import org.apache.axis2.description.Parameter;
+import org.apache.axis2.description.TransportOutDescription;
+import org.apache.axis2.engine.AxisConfiguration;
+import org.apache.log4j.BasicConfigurator;
+import org.testng.annotations.Test;
+
+public class FIXTransportSenderInitTest extends TestCase {
+    @Test()
+    public void testFIXTransportSenderInit() throws Exception {
+        AxisService axisService = new AxisService("testFIXService");
+        axisService
+                .addParameter(new Parameter(FIXConstants.FIX_ACCEPTOR_CONFIG_URL_PARAM, "/sample/path/Mock.cfg"));
+        axisService.addParameter(
+                new Parameter(FIXConstants.FIX_INITIATOR_CONFIG_URL_PARAM, "/sample/path/Mock2.cfg"));
+
+        ConfigurationContext cfgCtx = new ConfigurationContext(new AxisConfiguration());
+        TransportOutDescription trpOutDesc = new TransportOutDescription("fix");
+
+        FIXTransportSender fixTransportSender = new FIXTransportSender();
```

Comment:
We are not asserting anything?

Reply:
The test passes only if the sender is initialized. Therefore, we can say that if the sender is initialized, the test case passes (the other way around). Moreover, there's nothing to assert. Only the logs are there. Then we'll have to have a log-viewer client.